src/types/PortmanConstants.ts

```diff
@@ -0,0 +1,7 @@
+export const PortmanTestTypes = {
```

Comment:
just prefer this file to be named fromt he type `PortmanTestTypes`

Reply:
Perhaps we want to store also other constants in here? Or would you prefer a file per constant type?